external/wycheproof/README.md

```diff
@@ -0,0 +1,14 @@
+## Running External tests locally
```

Comment:
Please add a Heading and paragraph (similar to https://github.com/AdoptOpenJDK/openjdk-tests/pull/2452/files):

External Wycheproof Tests

Wycheproof tests are a part of the external third-party application tests that help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue #172 lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests. Wycheproof test material is pulled from the [wycheproof](https://github.com/google/wycheproof.git) repository.